recipe/meta.yaml

```
@@ -24,23 +25,54 @@ requirements:
     - {{ stdlib("c") }}
     - cmake >=3.25,<4
     - make  # [unix]
+
   host:
     - openssl
     - libabseil
   run:
     - libabseil
 
+outputs:
+
+  - name: s2geometry
+
+  - name: s2geometry-python
+    script: install_python.sh  # [unix]
+    script: install_python.bat  # [win]
+    build:
```

```suggestion
    build:
      script:
        - install_python.sh  # [unix]
        - install_python.bat  # [win]
```

Maybe this would fix the linter error?